From f837458a3b2e3947c7e685c54dffaa7ac6d9c6ae Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 21:13:48 -0500
Subject: [PATCH] Stop installing RPATHs

---
 rpm/ros-rolling-octomap.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ros-rolling-octomap.spec b/rpm/ros-rolling-octomap.spec
index 4f9117b966d1..19136b6b4dc5 100644
--- a/rpm/ros-rolling-octomap.spec
+++ b/rpm/ros-rolling-octomap.spec
@@ -63,6 +63,7 @@ on an octree. See http://octomap.github.io for details.
     -DCMAKE_PREFIX_PATH="%{bloom_prefix}" \
     -DSETUPTOOLS_DEB_LAYOUT=OFF \
     -DBUILD_TESTING=OFF \
+    -DCMAKE_SKIP_INSTALL_RPATH=ON \
 
 %cmake3_build
 
-- 
2.44.0

